Makefile

```diff
@@ -422,13 +422,13 @@ out/12-florida/state.gpkg: data/12-florida/statewide/2010/tl_2012_12_vtd10.zip
 # resume normal appends
 	ogr2ogr -sql "SELECT '2015' AS year, '12' AS state, '041' AS county, CONCAT('12041', Name) AS precinct, 'polygon' AS accuracy FROM FL_20121106v5_PctMap" \
 		-s_srs EPSG:4326 -t_srs EPSG:4326 -f GPKG -nln state -append \
-		$@ "data/12-florida/county/2016/GIL/GIL20121106v6_PctMap7f1098a5-00b8-4103-a5c6-c32c8d5d5cdc.kmz"
+		$@ "/vsizip/data/12-florida/county/2016/GIL/GIL20121106v6_PctMap7f1098a5-00b8-4103-a5c6-c32c8d5d5cdc.kmz"
```

Comment:
This path prefix doesn't make sense to me. Where is it coming from?

Reply:
http://www.gdal.org/gdal_virtual_file_systems.html#gdal_virtual_file_systems_vsizip

Reply:
Thanks, I learned something new about GDAL today :)